_docs-sources/refarch/access/how-to-auth-vpn/index.md

```diff
@@ -1,9 +1,44 @@
 # VPN Authentcation 
 
+Most of the AWS resources that comprise the Reference Architecture run in private subnets, which means they do not have a public IP address, and cannot be reached directly from the public Internet. This reduces the "surface area" that attackers can reach. Of course, you still need access into the VPCs, so we exposed a single entrypoint into the network: an [OpenVPN server](https://openvpn.net/).
 
-Haxx0r ipsum foo Trojan horse new all your base are belong to us ip error private shell fopen semaphore epoch char packet sniffer segfault gurfle bypass. Memory leak bubble sort injection leet malloc brute force double xss mega sudo mountain dew void echo win emacs linux piggyback bin. I'm compiling float bang case cat infinite loop Donald Knuth unix for /dev/null machine code then chown d00dz worm gnu crack packet bar eof while.
+You use an OpenVPN client to connect to the OpenVPN server, which gets you "in" to the network, allowing you to securely connect to private resources in the account.
 
-Lib void brute force bypass nak concurrently all your base are belong to us break leapfrog bit default packet sniffer Linus Torvalds. Man pages packet stack trace Starcraft Donald Knuth pwned worm hello world public giga frack gurfle. Irc fork malloc fopen script kiddies flood blob fail hexadecimal while access semaphore loop mega Trojan horse foo gobble.
+Here are the steps you'll need to take:
 
-Bang spoof *.* headers Dennis Ritchie pragma bubble sort mutex d00dz firewall wombat snarf. Win L0phtCrack back door big-endian tera injection flush suitably small values interpreter class hello world client segfault. Boolean buffer emacs highjack concurrently boolean I'm compiling malloc finally char protected void fopen ascii var cd Trojan horse public.
+1. [Install an OpenVPN client](#install-an-openvpn-client)
+1. [Join the OpenVPN IAM Group](#join-the-openvpn-iam-group)
+1. [Use openvpn-admin to generate a configuration file](#use-openvpn-admin-to-generate-a-configuration-file)
+1. [Connect to the OpenVPN server](#connect-to-the-openvpn-server)
 
+## Install an OpenVPN client
+
+There are free and paid OpenVPN clients available for most major operating systems:
+
+1. OS X: [Viscosity](https://www.sparklabs.com/viscosity/) or [Tunnelblick](https://tunnelblick.net/).
+1. Windows: [official client](https://openvpn.net/index.php/open-source/downloads.html).
+1. Linux: `apt-get install openvpn` or `yum install openvpn`.
+
+## Join the OpenVPN IAM Group
+
+Your IAM User needs access to SQS queues used by the OpenVPN server. Since IAM users are defined only in the security account, and the OpenVPN servers are defined in separate AWS accounts (stage, prod, etc), that means you need to "switch" to the accounts with the OpenVPN servers by assuming an IAM Role that has access to the SQS queues in those accounts.
```

Comment:
```suggestion
Your IAM User needs access to SQS queues used by the OpenVPN server. Since IAM users are defined only in the security account, and the OpenVPN servers are defined in separate AWS accounts (stage, prod, etc), that means you need to authenticate to the accounts with the OpenVPN servers by assuming an IAM Role that has access to the SQS queues in those accounts.
```